### DISCUSSION TITLE
v7.0.0 is not published to npm registry

### DISCUSSION THREAD

**Discussion by emzoumpo** [storybookjs/addon-designs#200] [Q&A] [answered]
### Affected design types

- [ ] figma
- [ ] figspec
- [ ] link
- [ ] image
- [ ] iframe

### Describe the bug

Somehow even though there's a release v7.0.0 which is also marked as "latest" it is not published to the npm registry: https://www.npmjs.com/package/storybook-addon-designs?activeTab=versions 

### How to reproduce the bug?

Go to npm: https://www.npmjs.com/package/storybook-addon-designs?activeTab=versions
There is no v7.0.0, despite the release being published on Github.

### Expected behaviour

I would expect that v7.0.0 would also be available to the npm registry, so the dependency can be updated.

### Environment

- [ ] Chrome / Chromium / Chromium based browsers
- [ ] Firefox
- [ ] macOS Safari
- [ ] iOS Safari
- [ ] Samsung Internet
- [ ] Other

### Affected versions

7.0.0

### Storybook versions

7.x.x

**Comment by pocka** [comment, marked as answer]
https://www.npmjs.com/package/@storybook/addon-designs

The package name is changed to `@storybook/addon-designs`

**Comment by thafryer** [comment]
~~Hi @emzoumpo! We published the new version at `@storybook/addon-designs`. You should see it in the npm registry there.~~

@pocka Can you run `npm deprecate` on the old package name? I could do it, but I do not have access to the original package.

https://docs.npmjs.com/cli/v9/commands/npm-deprecate

**Reply by pocka** [reply]
https://www.npmjs.com/package/@storybook/addon-designs

I think deprecation is not suitable for this situation (yet), as the new version drops SB6 compatibility.
Adding a note to README (like [this](https://github.com/pocka/storybook-addon-vue-info)) then publishing it as v6.3.2 would be better?

<img width="750" alt="Screenshot 2023-07-18 at 21 17 26" src="https://github.com/storybookjs/addon-designs/assets/13316015/c6823159-fcd7-4f17-a820-def3668a2889">

**Reply by emzoumpo** [reply]
If this is the best way to do it, I think there should also be a `7.0.0-beta.3` version that is doing the same (for reasons stated in my comment below: https://github.com/storybookjs/addon-designs/discussions/200#discussioncomment-6477810)

**Reply by pocka** [reply]
My bad, the web UI I was looking at is only for "package deprecation". The CLI command (`npm deprecate`) can deprecate specific version ranges. I'll mark `7.0.0-beta.2` as deprecated so users will be noticed.

**Reply by pocka** [reply]
Done.

https://www.npmjs.com/package/storybook-addon-designs/v/7.0.0-beta.2

**Comment by emzoumpo** [comment]
Hi @pocka @thafryer , thank you for the answer!

Since the package name was `storybook-addon-designs`, even on v7 beta (for example, in our `package.json` we have: `"storybook-addon-designs": "^7.0.0-beta.2"`), and that was the instructions for compatibility with Storybook v7 when in came out, so I assume a lot of people have a similar version in their dependencies, (since there was no stable release back then), it would be nice to have a deprecation warning for the old package.
